src/test/java/com/google/api/codegen/testdata/py/python_library.baseline

```diff
@@ -4300,16 +4300,12 @@ def sample_get_big_book():
 
   name = client.book_path('[SHELF_ID]', '[BOOK_ID]')
 
-  response = client.get_big_book(name)
+  operation = client.get_big_book(name)
 
-  def callback(operation_future):
-      # Handle result.
-      result = operation_future.result()
+  # Wait for the operation to finish, then return the result.
+  # Or use operation.add_done_callback to avoid blocking.
```

Comment:
@beccasaurus I think this doc can be better but I don't know how to improve it :P

Reply:
How about...

```py
def sample_get_big_book():
  """Testing calling forms"""

  client = library_v1.LibraryServiceClient()

  name = client.book_path('[SHELF_ID]', '[BOOK_ID]')

  operation = client.get_big_book(name)

  print('Waiting for operation to complete...')  # ⟸
  response = operation.result()

  print(response)
```

Short 'n sweet _( 'n [stolen](https://cloud.google.com/speech-to-text/docs/async-recognize#speech-async-recognize-gcs-python) from our existing Python samples_ 😛 _)_

Reply:
Nice